lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -110,6 +112,49 @@ def oauth_cannot_be_secure(manifest)
           end
         end
 
+        def invalid_scope_attribute_error(manifest)
+          errors = []
+          manifest.parameters.each do |parameter|
+            next if parameter.scopes.nil?
+            
+            errors << ValidationError.new(:field_contains_invalid_attributes, 
+                                        field: "parameters.[name=#{parameter.name}]", 
```

Comment:
I think the notation without `.` is closer to the array notation for JSON objects. This is kind of a hybrid between a JSON path and XPATH notation 😄. What do you reckon?

```suggestion
                                        field: "parameters[name=#{parameter.name}]", 
```